upstream/reconcile.go

```diff
@@ -56,12 +56,17 @@ func (t *upstreamReconciler) Sync(ctx context.Context, table string) error {
 func (t *upstreamReconciler) SyncAfter(ctx context.Context, table string, after time.Duration) error {
 	logger.WithValues("since", time.Now().Add(-after).Format(time.RFC3339Nano)).Debugf("Reconciling table %q with upstream", table)
 
-	// We find the item that falls just before the requested duration & begin from there.
-	var next string
-	if err := ctx.DB().Table(table).Select("id").Where("agent_id = ?", uuid.Nil).Where("NOW() - updated_at > ?", after).Order("updated_at DESC").Limit(1).Scan(&next).Error; err != nil {
-		return err
-	}
-
+	// We find the item that falls just before the requested duration & begin from there
+	//var next string
+	//if err := ctx.DB().Table(table).Select("id").Where("agent_id = ?", uuid.Nil).Where("NOW() - updated_at > ?", after).Order("id").Limit(1).Scan(&next).Error; err != nil {
+	//return err
+	//}
+	//if err := ctx.DB().Table(table).Select("id").Where("agent_id = ?", uuid.Nil).Where("NOW() - updated_at > ?", after).Order("id").Limit(1).Scan(&next).Error; err != nil {
+	//return err
+	//}
+
+	// We start with a nil UUID and calculate hash in batches
+	next := uuid.Nil.String()
 	return t.sync(ctx, table, next)
```

Comment:
I think we just get rid of `SyncAfter` since now it's identical to `Sync` method above it?

Reply:
I want to make sure this works before refactoring

Will clean it up once we decide to go this way 👍🏽